advanced/index.php

```diff
@@ -50,16 +50,24 @@ function setHeader($type = "x") {
 }
 
 // Determine block page type
-if ($serverName === "pi.hole") {
+if ($serverName === "pi.hole"
+    || (!empty($_SERVER["VIRTUAL_HOST"]) && $serverName === $_SERVER["VIRTUAL_HOST"])) {
     // Redirect to Web Interface
     exit(header("Location: /admin"));
 } elseif (filter_var($serverName, FILTER_VALIDATE_IP) || in_array($serverName, $authorizedHosts)) {
     // Set Splash Page output
     $splashPage = "
-    <html><head>
+    <html>
+      <head>
         $viewPort
-        <link rel='stylesheet' href='/pihole/blockingpage.css' type='text/css'/>
-    </head><body id='splashpage'><img src='/admin/img/logo.svg'/><br/>Pi-<b>hole</b>: Your black hole for Internet advertisements<br><a href='/admin'>Did you mean to go to the admin panel?</a></body></html>
+        <link rel='stylesheet' href='pihole/blockingpage.css' type='text/css'/>
+      </head>
+      <body id='splashpage'>
+        <img src='admin/img/logo.svg'/><br/>
```

Comment:
@mschoettle 
I just recognised that the blocking/404 page is broken when accessing a non-existent or blocked URL with a sub directory path, since the absolute asset paths have become relative with this commit. The client is then looking and failing for things like `http://blocked.domain.org/something/pihole/blockingpage.css`

Is there a reason why those need to be relative? Probably not that relevant with the planned removal of the blocking page, but the remaining 404 handler page still contains relative assets starting with our commit: https://github.com/pi-hole/pi-hole/pull/3910/files#diff-90be1003be891f4d6008bf3a167ceedcbc3e06dbf48e490c6bc761eef20ea96bR51

Reply:
There was a discussion in #3067 about relative paths and keeping it consistent. I guess because the block page is rarely seen (I have never seen it yet) this was not caught at the time. In this particular case it would make sense to not use relative URLs then.